### ISSUE TITLE
[Bug Report][2.6.4] Outlined text fields minimized label color issue

### ISSUE THREAD

**Issue by belov91** [vuetifyjs/vuetify#14932]
### Environment
**Vuetify Version:** 2.6.4
**Vue Version:** 2.6.14
**Browsers:** Chrome 97.0.4692.99
**OS:** Linux x86_64

### Steps to reproduce
Focus the field

### Expected Behavior
The label when focused should have a single background color.

### Actual Behavior
The label when focused has two background colors which make it unreadable and not good-looking.

### Reproduction Link
<a href="https://codepen.io/belov91/pen/rNpdpMe" target="_blank">https://codepen.io/belov91/pen/rNpdpMe</a>

**Comment by KaelWD** [comment]
How's that supposed to work? You can do `<template #label><span class="red">Name</span></template>` but it still looks like shit.

**Comment by belov91** [comment]
I'd suggest having some the same background with some rounded corners like this:
![image](https://user-images.githubusercontent.com/5794721/162161752-9e1b1e2a-5302-4bf1-a685-a721661067a1.png)

**Comment by lioneaglesolutions** [comment]
Yeah... I don't think this would ever look good - nor are there any example in MD3 showing usage like this. I would suggest using a different style of input 

Even any type of hack close to something like this will look horrible.
<img width="1221" alt="image" src="https://user-images.githubusercontent.com/61766491/235898934-b87a0ab8-f25b-41c9-a8ee-bec177053762.png">

I'd say this can be closed @johnleider
